### PR TITLE
fix/explorer-icons

### PR DESCRIPTION
**Pull request by bigboydiamonds**
## Summary by CodeRabbit

- **New Features**
	- Updated the display of blockchain explorer images and links.
- **Refactor**
	- Enhanced handling of transaction data in the `MostRecentTransaction` component.
- **Chores**
	- Added console log statements for debugging purposes.

e2a0265671dc954cce3a1b6dde459bd6c05c9c88: [synapse-interface preview link ](https://sanguine-synapse-interface-mm6gl4ip1-synapsecns.vercel.app)
9e49e01cd6a147c2a6d321e7e81be8d61fa2720a: [synapse-interface preview link ](https://sanguine-synapse-interface-ogzc2wose-synapsecns.vercel.app)
8741647344f758eb3259814f5b1b1fe65c1d1abd: [synapse-interface preview link ](https://sanguine-synapse-interface-mzj9ahokt-synapsecns.vercel.app)
dca5442dda9b6d0edccb0f50292f244d6f196090: [synapse-interface preview link ](https://sanguine-synapse-interface-1p4vraqwl-synapsecns.vercel.app)
eaecd72ac240271406becb2664441f0c7501ae97: [synapse-interface preview link ](https://sanguine-synapse-interface-i4r7cow59-synapsecns.vercel.app)
4382d21f1c5560ec9236ac1eb53246bca50ea1e0: [synapse-interface preview link ](https://sanguine-synapse-interface-g1xjh29jt-synapsecns.vercel.app)
7143cea36a819452eec22d50370e420896774ead: [synapse-interface preview link ](https://sanguine-synapse-interface-d564a5a7w-synapsecns.vercel.app)
03fd33bfaf8ad4a0ef788ffa286761218d2599dc: [synapse-interface preview link ](https://sanguine-synapse-interface-lc2ehysj9-synapsecns.vercel.app)
91baa6d136ce521005724d9141bf6d8ceff459da: [synapse-interface preview link ](https://sanguine-synapse-interface-aent5rdw8-synapsecns.vercel.app)
875afea83c5846b72b83407378e4e5a7a90155fc: [synapse-interface preview link ](https://sanguine-synapse-interface-786fnl52v-synapsecns.vercel.app)

### CODE REVIEW

**Comment by bigboydiamonds** [comment]
@coderabbitai pause

File: packages/synapse-interface/assets/explorer/terra.svg
Comment (abtestingalpha) [inline packages/synapse-interface/assets/explorer/terra.svg:1]: Don't need this anymore

File: packages/synapse-interface/constants/chains/master.tsx [hunk 1]
```diff
@@ -181,7 +181,7 @@ export const POLYGON: Chain = {
   nativeCurrency: { name: 'Matic', symbol: 'MATIC', decimals: 18 },
   explorerUrl: 'https://polygonscan.com',
   explorerName: 'PolygonScan',
-  explorerImg: polygonExplorerImg,
+  explorerImg: polygonImg,
```
Comment (abtestingalpha) [inline packages/synapse-interface/constants/chains/master.tsx:184]: We're using the SVG in `@assets/chains/...` here, meaning can we do that throughout so we don't have duplicates?

**Review by abtestingalpha — CHANGES_REQUESTED** [review]

File: packages/synapse-interface/constants/chains/master.tsx (hunk 1 above)
Comment (bigboydiamonds) [reply packages/synapse-interface/constants/chains/master.tsx]: Thanks for catching this! I was just testing out a few diff things to try and get the .png file to work. Didn't end up working so I converted all of `@assets/explorer/` files from .png to .svg

Switched back in https://github.com/synapsecns/sanguine/pull/1549/commits/dde17ade86d7d33a4d5af2e619fd7e2072d27c71

File: packages/synapse-interface/assets/explorer/terra.svg
Comment (bigboydiamonds) [reply packages/synapse-interface/assets/explorer/terra.svg:1]: removed in https://github.com/synapsecns/sanguine/pull/1549/commits/bb2f3ecf40617a46270301266661f8813a3ed266

**Review by abtestingalpha — APPROVED** [review]

**Pull request merged**